File: PHP/Route.php
<?php 
    class Route{
        private $id;
        private $start;
        private $end;

        public function __construct($start, $end)
        {
            $this->start = $start;
            $this->end = $end;
        }
        public function getId(){
            return $this->id;
        }
        public function getStart(){
            return $this->start;
        }
        public function getEnd(){
            return $this->end;
        }
        public function setStart($start){
            $this->start = $start;
        }
        public function setEnd($end){
            $this->end = $end;
        }
    }

?>

File: Python/Route.py
class Route:
    __id = int
    __start = []
    __end = []

    def __init__(self,start,end):
        self.__start = start
        self.__end = end
    
    def getId(self):
        return self.__id
    
    def setStart(self,start):
        self.__start=start
    
    def getStart(self):
        return self.__start
    
    def setEnd(self,end):
        self.__end=end
    
    def getEnd(self):
        return self.__end